docs/components/connectors/use-connectors/index.md

```diff
@@ -48,22 +48,54 @@ For further details on how secrets are implemented in Connectors, consult our [C
 Using this in other areas can lead to unexpected results and incidents.
 :::
 
-## Response mapping
+## Variable/response mapping
 
-Most Connectors have a `Response Mapping` section that typically consists of two fields: `Result Variable` and `Result Expression`. These fields are used to export responses from an external Connector call into process variables.
+In most instances, **Connectors** feature a dedicated section known as `Response Mapping`,
+comprising two essential fields: `Result Variable` and `Result Expression`.
+These fields export responses from external **Connector** calls into process variables.
 
-### Result Variable
+### Result variable
 
-This field declares a single process variable to export responses from a Connector call. You are able to use this process variable further in the process.
+This field declares a singular process variable designated to export responses from a **Connector** call.
+The resulting process variable can then be utilized within the ongoing process.
 
-### Result Expression
+### Result expression
 
-This field allows you to map a Connector response into multiple process variables which you are able to use further in the process. You can also transform the extracted values using [FEEL expressions](/components/concepts/expressions.md).
+This field facilitates the mapping of a **Connector** response into multiple process variables,
+providing further flexibility in their utilization within the ongoing process.
+Additionally, the extracted values can undergo transformation through the application of [FEEL expressions](/components/concepts/expressions.md).
```

Comment:
"undergo transformation" -- what does this mean exactly?

Reply:
Added